app/src/main/java/org/mozilla/fenix/library/LibraryFragment.kt

```diff
@@ -20,4 +25,29 @@ class LibraryFragment : Fragment() {
     ): View? {
         return inflater.inflate(R.layout.fragment_library, container, false)
     }
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setHasOptionsMenu(true)
+        val supportActionBar = (activity as AppCompatActivity).supportActionBar
+        supportActionBar?.show()
```

Comment:
nit: everywhere else we don't use the temporary variable. Let's keep it consistent :+1: